### PR TITLE
ROE-2108 Updating readme

### PR DESCRIPTION
**Pull request by markpit**
### JIRA link
https://companieshouse.atlassian.net/browse/ROE-2106

### Change description
Updating the Readme file for the auth-code-notification service

### Work checklist

- [ ] Tests added where applicable

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,26 @@
...
+### Requirements
+In order to run the service locally you will need the following:
+- [Java 11](https://www.oracle.com/java/technologies/downloads/#java11)
+- [Maven](https://maven.apache.org/download.cgi)
+- [Git](https://git-scm.com/downloads)
+
+### Getting started
+To check out and build the service:
+1. Clone [Docker CHS Development](https://github.com/companieshouse/docker-chs-development) and follow the steps in the README.
+2. Run ./bin/chs-dev modules enable overseas-entities
```
Comment (mattch1) [inline README.md:15]: Should this be auth-code-notification? If not could we add a note about dependency.

Comment (markpit) [reply README.md:15]: the docker file has been added under the overseas-entities module, i was wondering if we needed overseas-entities running as well for test data, but thinking about it we could have auth-code-notification in its own module. Another change would be needed to separate into its own module in the docker repo.

Comment (mouhajer-ch) [reply README.md:15]: It is part somehow of the update journey, but it could be separated if used with other services

File: README.md [hunk 2]
```diff
@@ -1,2 +1,26 @@
...
+3. Run ./bin/chs-dev development enable auth-code-notification (this will allow you to make changes).
+4. Run docker using "tilt up" in the docker-chs-development directory.
+5. Use space-bar in the command line to open tilt window - wait for auth-code-notification to become green.
+7. The service should be accessible using this url: http://chs.local/auth-code-notification
```
Comment (mouhajer-ch) [inline README.md:19]: The endpoint should be `http://api.chs.local/auth-code-notification/actuator` similar to all API service

**Pull request merged**